www/_posts/2023-09-12-camera-media-file-transfer-plugin-release.md

```diff
@@ -0,0 +1,218 @@
+
+    Furthermore, to support the new granular permissions and Android 13 (SDK 33), we have raised the minimum requirement for `cordova-android` to version `12.0.0`. This version of Cordova-Android specifically includes the necessary updates to handle Android 13 and compile your project with the new permissions successfully.
+
+* **Removed Deprecated Platforms**
+
+    As the Cordova-Windows and Cordova-OSX platform has been deprecated, the supporting logic for these platforms has been removed from this plugin.
+
+* **Retain Image Exif Data from Photo Library (iOS)**
+
+    This release contains a fix to preserve image's EXIF data for iOS.
+
+### `cordova-plugin-media`
+
+* **Android 13 Support**
+
+    In this release of the `media` plugin, the depdendency of the file plugin has been bumped to `8.0.0` which introduced Android 13 support. This support includes the Android 13's more granular permissions `READ_MEDIA_IMAGES`, `READ_MEDIA_VIDEO`, and `READ_MEDIA_AUDIO`.
+
+    Additional we have raised the minimum requirement for `cordova-android` to version `12.0.0`.
+
+    For more information, check out the [Cordova's File Plugin 8.0.0 release blog post](https://cordova.apache.org/news/2023/07/11/file-plugin-8.0.0.html).
+
+* **Removed Deprecated Windows Platforms**
+
+    As the Cordova-Windows platform has been deprecated, the supporting logic for that platform has been removed from this plugin.
+
+* **Ability to load files from custom scheme and leading slash directory paths**
+
+    The file plugin can now accept URL constructed with a custom scheme or a leading slash.
+
+    **Custom Scheme Example:**
+
+    By default, iOS uses the following custom scheme `app://localhost/`. You can now pass in `app://localhost/file.mp4` to prepresent a media file located in the root directory of which the app content is loaded from, `www`.
+
+    **Leading Slash Example:**
+
+    It can also load the same file `file.mp4` from the above example if the provided URL was `/file.mp4`. It will navigate from the root directory of which the app content is loaded from, `www`.
+
+* **Increased Android's Audio Quality**
+
+    From this plugin's version and onward, the Android's captured audio quality has been increased significantly. This was achieved by increasing the bit rate to 96 Kbps and the sampling rate to 44.1 kHz.
+
+    > **PR's Note:** This plugin uses the AAC encoder, which generally provides better quality audio at a lower bitrate compared against MP3 encoder. While researching acceptable bitrate of AAC compared against MP3, some suggest using 96 Kbps for AAC while MP3 would be 128 Kbps. But this does not mean it is identical. In terms of quality, it will always result in source and equiptment.
```

Comment:
```suggestion
    > **PR's Note:** This plugin uses the AAC encoder, which generally provides better quality audio at a lower bitrate compared against MP3 encoder. While researching acceptable bitrate of AAC compared against MP3, some suggest using 96 Kbps for AAC while MP3 would be 128 Kbps. But this does not mean it is identical. In terms of quality, it will always depend on the source and underlying equipment.
```